WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/detail/PostDayViewsUseCaseTest.kt

```diff
@@ -139,7 +141,6 @@ class PostDayViewsUseCaseTest : BaseUnitTest() {
     fun `manage edge condition with data available but empty list`() = test {
```

Comment:
I'll double check for you.

Reply:
Yes. Still valid the issue is unrelated to my changes. I only change the number of times `mapper` was called. Not related to the edge case. I haven't dug deeper if this could have been related. But I think it should be out of scope.

Reply:
Hmmm. You know what. I see your point. I didn't want to presume what the original issue was and that this fixes it. But based on the test, I agree. Will delete the test.